React/Base/RCTModuleMethod.mm

```diff
@@ -91,6 +91,7 @@ static BOOL RCTParseSelectorPart(const char **input, NSMutableString *selector)
 static BOOL RCTParseUnused(const char **input)
 {
   return RCTReadString(input, "__attribute__((unused))") ||
+         RCTReadString(input, "__attribute__((__unused__))") ||
```

Comment:
Works!

Reply:
Hi, may I know which file this is please? Thanks very much!